types/picomatch/picomatch-tests.ts

```diff
@@ -3,15 +3,20 @@ import pm = require("picomatch");
 new RegExp(/a/);
 // main function
 const isMatch = pm("*.!(*a)");
+// $ExpectType boolean
 isMatch("abcd");
+// $ExpectType boolean
+isMatch("abcd", true);
```

Comment:
So here an object should be returned not a boolean.

Reply:
Fixed.